Formula/source-to-image.rb

```diff
@@ -9,6 +9,7 @@ def install
   end
 
   test do
-    system "#{bin}/s2i", "-h"
+    system "#{bin}/s2i", "create", "testimage", testpath
+    assert_match(/Dockerfile/, shell_output("ls").chomp)
```

Comment:
You may want to use `assert (testpath/"Dockerfile").exist?` instead of parsing `ls`’s output.